ci-templates/github/model-complexity-badge/action.yml

```diff
@@ -0,0 +1,72 @@
+# SPDX-FileCopyrightText: Copyright DB Netz AG and the capellambse contributors
+# SPDX-License-Identifier: Apache-2.0
+
+# SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+# SPDX-License-Identifier: Apache-2.0
+
+name: 'Generate Model Complexity Badge'
+description: 'An action to create and push a model complexity badge'
+
+inputs:
+  commit_msg:
+    description: 'The commit message for updating the model complexity badge'
+    required: false
+    default: "docs: Update model complexity badge"
+  output_file:
+    description: 'The file path to store the generated model badge'
+    required: false
+    default: model-complexity-badge.svg
+  entry_point:
+    description: 'The entry point of the used model'
+  capellambse_revision:
+    description: 'The revision of the py-capellambse package to install'
+    required: false
+    default: master
```

Comment:
I will make it like that - but is somebody going to remember to change that?

Reply:
We'll just do a grep for `0\.5` through the repo once 1.0 lands. That will do :)

Reply:
alright :D

Reply:
Correction (I only just saw the actual edit you made):

I didn't want to hardcode the *current* "latest stable release", I wanted it to use the literal `release/0.5` ref. That branch tracks the latest stable release, so that we have a simple (and automatically updated) way of referring to it.